uint64 cmdMask
uint8 autoFocusMode
uint8 lensPosition
ManualExposureParams expManual
RegionParams aeRegion
RegionParams afRegion
uint8 awbMode
uint8 sceneMode
uint8 antiBandingMode
uint8 effectMode
bool aeLockMode
bool awbLockMode
int8 expCompensation
int8 brightness
int8 contrast
int8 saturation
uint8 sharpness
uint8 lumaDenoise
uint8 chromaDenoise
uint8[] data
